product_plm_import/wizards/product_plm_import.py

```diff
@@ -22,6 +22,7 @@
     ["item_type", "Item Type", "char", True],
     ["category", "Category", "char", True],
     ["uom", "Unit of Material", "char", False],
+    ["quality_check_categ_id", "Quality Check Category", "char", False],
```

Comment:
Following the request from the client. Never have `_id` in the variable name unless it really is an ID.

```suggestion
    ["acceptance_test_categ", "Acceptance Test Category", "char", False],
```